RadixWallet/Features/InteractionReview/Sections/Sections+ManifestSummary.swift

```diff
@@ -1,6 +1,242 @@
+import Sargon
+
 extension InteractionReview.Sections {
 	func sections(for summary: ManifestSummary, networkID: NetworkID) async throws -> Common.SectionsData? {
-		// TODO: Implement
-		nil
+		let allWithdrawAddresses = summary.accountWithdrawals.values.flatMap { $0 }.map(\.resourceAddress)
+		let allDepositAddresses = summary.accountDeposits.values.flatMap(\.specifiedResources).map(\.resourceAddress)
+
+		// Pre-populate with all resource addresses from withdraw and deposit.
+		let allAddresses: IdentifiedArrayOf<ResourceAddress> = Array((allWithdrawAddresses + allDepositAddresses).uniqued()).asIdentified()
+
+		func resourcesInfo(_ resourceAddresses: [ResourceAddress]) async throws -> ResourcesInfo {
+			try await onLedgerEntitiesClient.getResources(resourceAddresses)
+				.reduce(into: ResourcesInfo()) { partialResult, next in
+					partialResult[next.resourceAddress] = .left(next)
+				}
+		}
+
+		let resourcesInfo = try await resourcesInfo(allAddresses.elements)
+
+		let withdrawals = try await extractWithdrawals(
+			accountWithdraws: summary.accountWithdrawals,
+			entities: resourcesInfo,
+			networkID: networkID
+		)
+
+		let deposits = try await extractDeposits(
+			accountDeposits: summary.accountDeposits,
+			entities: resourcesInfo,
+			networkID: networkID
+		)
+
+		let dappAddresses = summary.encounteredEntities.compactMap {
+			switch $0 {
+			case let .component(componentAddress):
+				componentAddress.isGlobal ? componentAddress.asGeneral : nil
+			case let .locker(lockerAddress):
+				lockerAddress.asGeneral
+			}
+		}
```

Comment:
This should be common with `sections(for summary: ExecutionSummary,`, worth adding an extension to be used in both places.